src/west/app/project.py

```diff
@@ -89,6 +89,7 @@ def _handle_failed(self, args, failed):
         # and report errors if anything failed.
 
         if not failed:
+            self.dbg(f'{self.name} failed for zero project')
```

Comment:
I don't understand this message?

Reply:
This shows "diff failed for zero project" and will never be printed unless there's a bug in `west diff`.

It may print "status failed for zero project", same for "update" and maybe other commands.

How about this small change:

```suggestion
            self.dbg(f'git {self.name} failed for zero project')
```